src/components/Search/ResultItem/ResultItem.jsx

```diff
@@ -42,6 +42,9 @@ const messages = defineMessages({
 });
 
 const Marker = ({ text = '', highlight = '' }) => {
+  if (!text) {
+    return '';
+  }
```

Comment:
questo secondo me non serve perchè il default è sempre stringa vuota (vedi riga 44)

Reply:
eh no perché se gli viene passato come null, il default non conta.
Il default entra in gioco solo se non passi il parametro text